viewer/src/components/BoardAction.vue

```diff
@@ -73,7 +75,16 @@ export default class BoardAction extends Vue {
   }
 
   get faded() {
-    return !this.$store.state.gaiaViewer.data.boardActions[this.action];
+    return this.planet != null;
+  }
+
+  get planet(): Planet {
+    const data = this.$store.state.gaiaViewer.data;
+    const player = data.boardActions[this.action];
+    if (player != null) {
+      return factions.planet(data.player(player).faction);
+    }
```

Comment:
To keep compatibility with ongoing & ended games, the viewer should:

- continue to handle the case `typeof player === 'boolean'`
- handle if `player === Player.Player5` (see comment about `Engine.fromData`)